test/plugins/individual-collateral/compoundv3/CusdcV3Wrapper.test.ts

```diff
@@ -639,15 +641,17 @@ describeFork('Wrapped CUSDCv3', () => {
       await advanceBlocks(1)
 
       expect(await compToken.balanceOf(bob.address)).to.be.greaterThan(0)
-      expect(await compToken.balanceOf(bob.address)).to.equal(
-        await compToken.balanceOf(don.address)
-      )
+      const expectedBalanceBob = await compToken.balanceOf(bob.address)
```

Comment:
why do we have to loosen this check?

Reply:
There is a small difference i the numbers. Maybe due to logic based on blocks/network. It was 1 atto token in one case and less than 0.5% in another (maybe less even).